src/AspNet.Security.OAuth.Validation/Events/BaseValidationContext.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace AspNet.Security.OAuth.Validation.Events
+{
+    public abstract class BaseValidationContext
```

Comment:
This base class is not necessary, as aspnet/Security already offers a similar one that does the same thing: https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication/Events/BaseControlContext.cs#L8